test/Microsoft.OpenApi.Readers.Tests/Microsoft.OpenApi.Readers.Tests.csproj

```diff
@@ -65,6 +65,21 @@
         <None Update="V2Tests\Samples\minimal.v3.yaml">
             <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
         </None>
+        <None Update="V2Tests\Samples\OpenApiParameter\formDataParameter.yaml">
+          <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
Can we embed this files with the test.dll?

Reply:
I'm not sure about the pros and cons of this. @darrelmiller also pointed this out earlier.

What's the disadvantage of not embedding a test file?

Reply:
@PerthCharern It just means you have to carry those files around to make the tests work.  If they are embedded in the DLL then the tests are self-contained.

Reply:
I think I understand the concerns now. At first I thought using a non-embedded resource with relative path should work, but that might not be the case since the test assembly can technically be run from anywhere. 

Let's make this change in a separate PR for all resources.